iTaylorCodes/Gamr
# Users should be able to match with other users

### What does this PR do?
- This Pull Request implements the match making feature.
- This PR closes #22 
### Description of Task to be completed
- Users should be able to see a homepage where they can match with other users.
### How should this be manually tested?
- Login and visit the homepage. Verify it displays a user with the option to match profiles or skip. Match with a user and the user should not be displayed again.

https://user-images.githubusercontent.com/76837421/131778975-db1497d0-8a9a-4750-a917-cbde616b2221.mov
